examples/step-19/doc/results.dox

```diff
@@ -159,77 +159,6 @@ electrode.
 <a name="extensions"></a>
 <h3>Possibilities for extensions</h3>
 
-<h4> Avoiding a performance bottleneck with particles </h4>
```

Comment:
It's nice that we can now remove this section. Wasn't there an in-code comment pointing to this section and the related bottleneck?

Reply:
There was a comment in the documentation of the particles_in_cell() function, but I removed that as well.